6주차/14938/14398_python_토르.py

```diff
@@ -0,0 +1,50 @@
+from collections import deque
+import sys
+import copy
+from collections import defaultdict
+import heapq
+
+# 지역의 개수 n, 수색 범위 m, 길의 개수 r
+n, m, r = map(int, input().split())
+
+items = list(map(int, input().split()))
+graph = [[] for _ in range(n)]
+
+answer = 0
+
+def dijkstra(start_node):
+    global answer
+    distances = [sys.maxsize] * len(items)
+    distances[start_node] = 0
+    priority_queue = []
+    heapq.heappush(priority_queue, (0, start_node))
+
+    while priority_queue:
+        cost, node = heapq.heappop(priority_queue)
+
+        for adj_node_cost in graph[node]:
+            adj_cost = adj_node_cost[0]
+            adj_node = adj_node_cost[1]
```

Comment:
```suggestion
        for adj_cost, adj_node in graph[node]:
```
일케 뽑아올 수 있지 않나요? (아닌가)